pom.xml

```diff
@@ -31,12 +31,19 @@
     <jmhjar.name>benchmarks</jmhjar.name>
     <tycho-version>1.2.0</tycho-version>
     <cbi-plugins.version>1.1.5</cbi-plugins.version>
-    <surefire.version>2.22.0</surefire.version>
     <junit.version>5.3.0</junit.version>
-    <maven-plugin-tools.version>3.5.2</maven-plugin-tools.version>
     <maven.version>3.5.0</maven.version>
     <unix.socket.tmp></unix.socket.tmp>
 
+    <!-- some maven plugins versions -->
+    <surefire.version>2.22.0</surefire.version>
```

Comment:
Can we make this have the same format of the others, i.e. `maven.surefire.plugin.version`?